.github/ISSUE_TEMPLATE/release.md

```diff
@@ -1,86 +1,75 @@
 ---
-name: Publish a release
+name: Publish a major or minor release
 about: Publish a new Orchard Core release
 title: 'Release v'
 labels: release
 assignees: ''
 
 ---
-<!-- Be sure to also read https://docs.orchardcore.net/en/latest/topics/publishing-releases/. While the checklist is in a recommended order not every step depends strictly on the previous ones.  
```

Comment:
Why remove this?

Reply:
It's not needed as we documented exactly what we need. Extra noise for 0 added value.

Reply:
Semantic versioning is explained only there. We could also cut down on the repetition by moving e.g. the Version Updates Checklist there and only linking to there instead of having the same bullet points twice (still have a checkbox, but only like "Version Updates Checklist, see the documentation", and have the long text only there).

Reply:
Because you know what's on the page. I'm sure you weren't fully attentive with most of the other points either, since you already knew what to do. However, this doesn't mean that others wouldn't benefit from it, and the point of documentation is to have something that others can utilize too.

Reply:
I get that. But the steps in this issue are the only steps one need to do to successfully releasing a new version "even if this is their first time doing it".

Reply:
Where does it explain how you determine the next version number? ;)

Reply:
That depends what are you releasing. But it's common sense that the next release is adding 1 to the patch, minor or major. If you can't decide what should the next version be, you not be doing the release to being with. ;)

Reply:
Alright :).